### PR TITLE
Update CORS.md link from cors.js to security.js

### PR DESCRIPTION
**Pull request by mdfrederick**
From 0.X to 1.X the configuration for CORS has moved from cors.js to security.js

### CODE REVIEW

File: concepts/Security/CORS.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Sails can be configured to allow cross-origin requests from a list of domains yo
 
 For security reasons, CORS is disabled by default in Sails.  But enabling it is dead-simple.
 
-To allow cross-origin requests from a whitelist of trusted domains to _any_ route in your app, simply enable `allRoutes` and provide an `origin` setting in [`config/cors.js`](https://sailsjs.com/docs/reference/configuration/sails-config-cors):
+To allow cross-origin requests from a whitelist of trusted domains to _any_ route in your app, simply enable `allRoutes` and provide an `origin` setting in [`config/security.js`](https://sailsjs.com/docs/reference/configuration/sails-config-cors):
```
Comment (mikermcneil) [inline concepts/Security/CORS.md]: Thanks! Just for consistency would you change the URL as well? There’s a redirect set up, but might as well change it too

**Comment by mdfrederick** [comment]
Would you prefer the more generic `https://sailsjs.com/documentation/reference/configuration/sails-config-security` or the more direct `https://sailsjs.com/documentation/reference/configuration/sails-config-security#?sailsconfigsecuritycors`?

**Comment by mikermcneil** [comment]
@mdfrederick w/ fragment is good -- only thing is that a link to a particular can break b/c we don't have the resources to maintain target-level permalinks, but even if the target changes, it's ok because at least the page will be correct

**Comment by mikermcneil** [comment]
@mdfrederick thanks!

**Pull request merged**